frontend/src/components/common/EditModuleModal.tsx

```diff
@@ -54,17 +66,51 @@ const EditModuleModal = ({
       );
       return;
     }
-    const newModule = { ...module, title, description, published: isPublished };
-    const [id, course] = formatCourseRequest(newModule);
-    updateCourse({
-      variables: { id, course },
-      refetchQueries: [
-        { query: COURSES },
-        { query: GET_COURSE, variables: { id } },
-      ],
-    });
-    setInvalid(false);
-    onClose();
+
+    if (previewImage) {
+      const newModule = {
+        ...module,
+        title,
+        description,
+        published: isPublished,
+        previewImage,
```

Comment:
Instead of setting `previewImage` to the actual preview URL here (which expires after 60 minutes), I'm thinking it would be good to set it to the file path, and hydrate `filePath -> previewImage` later during `GET_COURSE` or `COURSES` queries. That's because currently the image URLs break after that hour passes. Thoughts?